### ISSUE TITLE
Missing annotation files in the OVVG dataset

### ISSUE THREAD

**Issue by JierunChen** [cv516Buaa/OV-VG#3]
Hi, thanks for the dataset contribution. However, it seems that the annotation files are not provided in the OVVG zip file. Could you double-check on that?

**Comment by chaishare** [comment]
Indeed, I also couldn’t find the annotation files in the OV-VG dataset.
